src/common/extension-telemetry-events.ts

```diff
@@ -170,6 +170,28 @@ export type ValidatePortTelemetryData = {
     port: number;
 };
 
+export type AndroidScanCompletedTelemetryData = {
+    port: number;
+    scanDuration: number;
+} & InstanceCount;
+
+export type InstanceCount = {
+    PASS: {
```

Comment:
have you made this uppercase to be consistent with web telemetry data type?

Reply:
Not really, this is the casing we get from axe-android service